### PR TITLE
Fix logger examples (great migration)

### PR DESCRIPTION
**Pull request by JeffLIrion**
**Description:**

Fix and improve examples for the logger component. 

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

Generated documentation: https://deploy-preview-9567--home-assistant-docs.netlify.com/components/logger/

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by JeffLIrion** [comment]
@Swamp-Ig here is the pull request.

**Comment by Swamp-Ig** [comment]
Hey, sorry I didn't mean to clobber your changes. I was trying to do a review but that's how it came out. 

Just a proposal to simplify the documentation a bit.

I don't think it makes a lot of sense to even include `homeasssistant.components.media_player` and the like, and with an integration it probably makes sense just to include the whole thing, rather than a sub-package.

**Comment by JeffLIrion** [comment]
I think it's worthwhile to show how you can set the log level for one component (`media_player`) in an integration to be different from that of the integration as a whole (`yamaha`). But this should probably be an integration other than `yamaha`, which only has one component.

**Comment by JeffLIrion** [comment]
How about this?

```yaml
logger:
  default: error
  logs:
    # log level for HA core
    homeassistant.core: fatal

    # log level for all lights
    homeassistant.components.light: warning

    # log level for SmartThings lights
    homeassistant.components.smartthings.light: info

    # log level for a custom component
    custom_components.my_integration: debug

    # log level for the `aiohttp` Python package
    aiohttp: critical
```

**Comment by Swamp-Ig** [comment]
The log level for all lights bit won't work though. It will only maybe cover the infrastructure code and possibly the base type.

**Comment by JeffLIrion** [comment]
That's basically the same as what is currently in the documentation:

```yaml
logger:
  logs:
    homeassistant.components.device_tracker: critical
    homeassistant.components.camera: critical
```

Are you saying that those examples wouldn't work?

**Comment by Swamp-Ig** [comment]
Nope, not in the way you're expecting anyhow.

File: source/_components/logger.markdown [hunk 1]
```diff
@@ -45,11 +45,17 @@ components:
...
   logs:
-    homeassistant.components: info
-    homeassistant.components.rfxtrx: debug
-    homeassistant.components.lifx: warning
-    homeassistant.components.yamaha: debug
+    # log level for HA core
+    homeassistant.core: fatal
+    
+    # log level for SmartThings lights
+    homeassistant.components.smartthings.light: info
```
Comment (Swamp-Ig) [inline source/_components/logger.markdown:55]: Maybe just `homeassistant.components.smartthings`? Put lights below if needs be. Most likely you'd log a whole integration, not just a small part of it.

Comment (JeffLIrion) [reply source/_components/logger.markdown:55]: How about this latest change?  This demonstrates how you can set the log level for an integration (MQTT) and also for a component (SmartThings light).

Comment (Swamp-Ig) [reply source/_components/logger.markdown:55]: Looks good to me :grin:

**Review by Swamp-Ig — APPROVED** [review]
I'd be happy to approve for myself, but will let someone else have the final say.

**Comment by JeffLIrion** [comment]
Anyone willing to merge this? Any additional changes I should make?

**Review by frenck — APPROVED** [review]
Looks good to me! Thanks! 👍

**Pull request merged**